### PR TITLE
[WIP] Feature/z prop power

### PR DESCRIPTION
**Pull request by mammykins**

### CODE REVIEW

**Comment by mammykins** [comment]
Completed power analysis of z prop test

**Comment by suganya-s** [comment]
the rationale looks good 👍

File: z_prop_test_power_analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+* Current Rate or baseline Related Link Click rate  
+* Effect size or Minimum Detectable Change (the minimum change you want to be able to detect)  
+* Statistical Significance: the probability of mistakenly rejecting the null hypothesis (H~0~) if it were true  
+* Statistical Power: probability of correctly rejecting the null hypothesis (H~0~) when the alternative (H~1~) is true. In other words, the ability of a test to detect an effect if the effect actually exists.  
+
+We can do this in R easily enough:  
+
+How many user journey sessions (n) do we need per page variant to be able to detect a 5% relative difference in related link click through rate (p~A~)? Let’s take the most extreme example to ensure we have sufficient power.  
+
```
[the remark this answers is not in the document]
Comment (mammykins) [reply z_prop_test_power_analysis.Rmd]: yep

[the remark this answers is not in the document]
Comment (mammykins) [reply z_prop_test_power_analysis.Rmd]: hmm forget to spellcheck

File: z_prop_test_power_analysis.Rmd [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+If X is the number of successes and n is the number of trials then, then p is the probability of success. The probability of failure, is q or 1 - p. So for metric 2a, a success would be a journey (‘Sequence’ in the data pipeline language) with at least one related link clicked during it, multiplied by the number of occurrences of that journey. The number of trials would be the sum of journeys multiplied by their respective occurrences.  
+
+pA = XA / nA  
+pB = XB / nB  
+
+The simplest and most frequently used modelling approach is to aggregate all the data together by page variant (as above). For comparison of these two observed proportions we can use the two-proportions z-test . We want to know, whether the proportions of clickers are the same in the two groups of user journeys.  
+
+The test statistic can be calculated as follows:  
+
+$$z = \frac{(p_A - p_B) - 0} {\sqrt{(pq) (\frac{1}{n_A} + \frac{1}{n_B})}}$$
```
Comment (suganya-s) [inline z_prop_test_power_analysis.Rmd]: We could maybe add the general formula for the test statistic here/in an explanatory doc during firebreak to show how we got the 0, pq, etc.

Comment (mammykins) [reply z_prop_test_power_analysis.Rmd]: 👍

**Pull request merged**